library/openssl/digest/reset_spec.rb

```diff
@@ -0,0 +1,38 @@
+require_relative '../../../spec_helper'
+require_relative '../../../library/digest/sha1/shared/constants'
+require_relative '../../../library/digest/sha256/shared/constants'
+require_relative '../../../library/digest/sha384/shared/constants'
+require_relative '../../../library/digest/sha512/shared/constants'
+require 'openssl'
+
+describe "OpenSSL::Digest#reset" do
+  describe "resets the state of the digest" do
```

Comment:
**minor:**

```suggestion
  describe "resetting the state of the digest" do
```

But TBH I would avoid a nested describe as far as it's the only nested section.